Makefile

```
@@ -2,6 +2,8 @@ BUILDER ?= virtualbox-iso.virtualbox
 VERSION ?= 22.05
 ARCH ?= x86_64
 REPO ?= nixos/nixos
+BUILD_PROVIBER = $(word 2, $(subst ., ,${BUILDER}))
```

```suggestion
BUILD_PROVIDER = $(word 2, $(subst ., ,${BUILDER}))
```
There is still one character typo. You also need to update all the places where it's being used.

---

No worries, but the typo is still there. PROVI**D**ER instead of PROVI**B**ER